webview/src/experiments/components/table/styles.module.scss

```diff
@@ -998,3 +998,7 @@ $badge-size: 0.85rem;
   font-size: 0.65rem;
   cursor: pointer;
 }
+
+.addConfigButton {
```

Comment:
Would it look better if the button was directly below the table instead of stuck at the bottom of the webview? 

<img width="1418" alt="Screenshot 2023-02-16 at 7 54 06 AM" src="https://user-images.githubusercontent.com/43496356/219383634-604e725c-786e-4d73-aa40-f577b96a74d0.png">

Reply:
You're right, changed.